frontend/config/webpack.config.prod.js

```diff
@@ -0,0 +1,346 @@
+const autoprefixer = require('autoprefixer');
```

Comment:
Please remove this file since `webpack.config.prod.js` was replaced by `webpack.config.js` in #102